rails/benchmarks/activemodel_views.rb

```diff
@@ -37,7 +39,7 @@ def user_factory
   { email: Faker::Internet.email }
 end
 
-def render_views
+m = Benchmark.rails(100, "actionview/render_activemodels") do
```

Comment:
Ah ok I just saw how `Benchmark::rails` is defined. We should use the keys explicitly for readability :)